src/app/components/search/search.component.html

```diff
@@ -1,48 +1,90 @@
 <!-- Start of form -->
 <form [formGroup]="form" (ngSubmit)="onSubmit()">
-  <div class="govuk-form-group">
-    <h1 class="govuk-label-wrapper">
-      <label class="govuk-label govuk-label--l" for="case-search"> Search for a case </label>
-    </h1>
-    <label class="govuk-label" for="case-search"> Case ID </label>
-    <div id="case-search-hint" class="govuk-hint">
+  <h1 class="govuk-label-wrapper">
+    <label class="govuk-label govuk-label--l" for="case-search"> Search for a case </label>
+  </h1>
+
+  <div *ngIf="errorSummary.length > 0" class="govuk-error-summary" data-module="govuk-error-summary">
+    <div role="alert">
+      <h2 class="govuk-error-summary__title">There is a problem</h2>
+      <div class="govuk-error-summary__body">
+        <ul class="govuk-list govuk-error-summary__list">
+          <li *ngFor="let error of errorSummary">
+            <a href="search#{{ error.fieldId }}">{{ error.message }}</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
+
+  <div class="govuk-form-group" [ngClass]="{ 'govuk-form-group--error': isSubmitted && f.case_number.errors }">
+    <label class="govuk-label" for="case_number"> Case ID </label>
+    <div id="case_number-hint" class="govuk-hint">
       Also known as a case reference or court reference. There should be no spaces.
     </div>
+    <div *ngIf="isSubmitted && f.case_number.errors">
+      <p *ngFor="let error of getFieldErrorMessages('case_number')" id="casenumber-error" class="govuk-error-message">
+        <span class="govuk-visually-hidden">Error:</span> {{ error }}
+      </p>
+    </div>
     <input
       formControlName="case_number"
-      required
       minlength="1"
       class="govuk-input govuk-input--width-10"
-      id="case-search"
-      name="case-search"
+      id="case_number"
+      name="case_number"
       type="text"
       aria-describedby="case-search-hint"
+      aria-describedby="case_number-hint"
+      [ngClass]="{ 'govuk-input--error': isSubmitted && f.case_number.errors }"
     />
   </div>
-  <details class="govuk-details" data-module="govuk-details">
-    <summary class="govuk-details__summary">
+  <details class="govuk-details" data-module="govuk-details" [open]="isAdvancedSearch">
+    <summary class="govuk-details__summary" (click)="toggleAdvancedSearch($event)">
```

Comment:
Is passing the event through a standard/normal thing to do in angular?

Reply:
Yes it is. Had to prevent the default behaviour of the event from propagating so we can control the open state of the details element